features/dynamic-bundle-discounts.md

```diff
@@ -0,0 +1,210 @@
+# Dynamic Bundle discounts
+
+- [Dynamic Bundle discounts](#dynamic-bundle-discounts)
+  - [Introduction](#introduction)
+  - [Setup](#setup)
+    - [Nosto Dynamic Bundle template](#nosto-dynamic-bundle-template)
+    - [Shopify Line Item script](#shopify-line-item-script)
+    - [Known Issues](#known-issues)
+
+## Introduction
+This documentation explains the process of setting up `Nosto - Dynamic Bundles` for automatic discount, when a bundle is added to the cart on a Shopify store. As per the existing setup, the end-to-end process has to be done manually. This release improves this by reducing manual intervention, to an extent, in setting up the store for bundle discounts.
+
+## Setup
+
+### Nosto Dynamic Bundle template
+A new Nosto JavaScript API `addBundleToCartWithDiscount` has been introduced. Functionalities of this API are as follows, 
+- collect the bundle information, from the bundle template, 
+- **validate and prepare the data for Shopify API call** (This step adds discount details along with bundle information to line item properties)
+- invoke Shopify API with the data
```

Comment:
I think this and the previous point can be consolidated into:
- Adds the products to the cart using [Shopify's cart API](https://shopify.dev/api/ajax/reference/cart).

Reply:
Why do they need to know the Shopify Cart API details here. It's just a informative document on what the JavaScript API does internally. I think providing the link would be too diverting